source/partials/next_release/_release.md.erb

```diff
@@ -1,73 +1,49 @@
-
-    Pipeline instances have an explict link to VSM across the application for better discoverablity. Moreover, the term pipeline `Label` has been changed to `Instance` as part of an ongoing UX refresh.
-
-  <figure class='shrink-figure-50'>
-      <%= image_tag 'releases/17.11.0/vsm-explicit-link-dashboard.png', alt: 'Explicit vsm link in dashboard' %>
+<figure class='shrink-figure-75'>
+      <%= image_tag 'releases/17.12.0/Plugins-SPA.png', alt: 'Plugins SPA' %>
   </figure>
+* <%= link_to_commit '979fa507d3527927919a730bf421e92c2090a972', 'Addition of pipeline count information to about page', 3960%>
 
-  <figure class='shrink-figure-50'>
-      <%= image_tag 'releases/17.11.0/vsm-explicit-link-vsm.png', alt: 'Explicit vsm link in vsm' %>
-  </figure>
+<h4>Other Improvements</h4>
+* <%= link_to_commit '0ee52b367080b758847613c21701c11895e8ba4a', 'Disallow an authenticated user from re-authentication', 3940%>
 
-  Page headers refreshed across application
-  <figure class='shrink-figure-50'>
-      <%= image_tag 'releases/17.11.0/vsm-explicit-link-stage-details.png', alt: 'Explicit vsm link in stage details header' %>
-  </figure>
+     With this change, an authenticated user is re-directed to home page on trying to access the login page.
+* <%= link_to_commit '6fda2f5cb5104b9394f2cfd5e41ca1e517078bd6', 'Add new unlock behavior', 3943%> <%= link_to_issue 106 %>
 
-  <figure class='shrink-figure-50'>
-      <%= image_tag 'releases/17.11.0/vsm-explicit-link-vsm-header.png', alt: 'Explicit vsm link in vsm header' %>
-  </figure>
+    As a part of this change, two types of locking behavior are introduced:
 
-  <figure class='shrink-figure-50'>
-      <%= image_tag 'releases/17.11.0/vsm-explicit-link-job-details-header.png', alt: 'Explicit vsm link in job details page header' %>
-  </figure>
+    * Run single instance of pipeline at a time ("Lock behavior: Unlock when finished")
 
-<h4>Other Improvements</h4>
-
-* <%= link_to_commit 'eb7305e41c03df0266974c6b7f432aac4f491cf1', 'Improves classloading and resource management', 3792 %>
+    * Run single instance of pipeline and lock on failure ("Lock behavior: Lock on failure")
 
-  Prevents file-descriptor and classloader leaks by ensuring that:
+    More information can be found in the <%= link_to("docs", "https://docs.gocd.org/17.12.0/configuration/admin_lock_pipelines.html") %>
 
-  * Classloaders for agent-launcher and agent are closed properly
-  * Temporary jar files that are extracted are removed after closing the
-    classloaders
-  * File watcher from log4j is replaced with a custom implementation
+<h4>Bug Fixes</h4>
+* <%= link_to_commit 'ab1ccf83aa90dfa3fc916ca892592e6db81717fc', 'Update server side validation to not use file system', 4026%> <%= link_to_issue 3984 %>
+* <%= link_to_commit 'e959f9d436e6c01ae61d080bd37b68117c7076d0', 'Fix multiple logs showing on build details page\'s console tab', 4024%> <%= link_to_issue 4018 %>
```

Comment:
Use double quotes so as to not have to escape the single quote inside?

Reply:
@yankurk -- this issue still persists. Do not add this as part of release notes.